templates/index.html

```diff
@@ -38,390 +41,523 @@ <h2 class="p-heading--two">Say hello to your physical&nbsp;cloud</h2>
-            </div>
-            <p>Monitors and tracks critical services to ensure proper operations.</p>
-          </div>
-        </li>
-        <li class="p-matrix__item">
-          <div class="p-heading-icon">
-            <div class="p-heading-icon__header">
-              <img class="p-heading-icon__img" src="{{ ASSET_SERVER_URL }}81ca0037-manage.svg" alt="Manage icon" />
-              <h4 class="p-heading-icon__title"><a class="p-link--strong p-heading--five" href="/tour#hardware-tracking" onClick="dataLayer.push({'event' : 'GAEvent', 'eventCategory' : 'Feature', 'eventAction' : 'Click', 'eventLabel' : 'Service tracking' });"><span>Manage</span>&nbsp;&rsaquo;</a></h4>
-            </div>
-            <p>Comes with a <a aria-label="External link to go to MAAS API docs" href="https://docs.maas.io/en/api" onClick="dataLayer.push({'event' : 'GAEvent', 'eventCategory' : 'Feature', 'eventAction' : 'Click', 'eventLabel' : 'REST API, CLI' });" class="p-link--external">REST <abbr title="Application Programming Interface">API</abbr></a>, Web <abbr title="User Interface">UI</abbr> and <a aria-label="External link to go to MAAS CLI docs" href="https://docs.maas.io/maas/en/manage-cli" onClick="dataLayer.push({'event' : 'GAEvent', 'eventCategory' : 'Feature', 'eventAction' : 'Click', 'eventLabel' : 'REST API, CLI' });" class="p-link--external"><abbr title="Command Line Interface">CLI</abbr></a>.</p>
-          </div>
-        </li>
-      </ul>
+    <div class="col-6">
+      <h4>Carrier-grade infrastructure DNS, DHCP and IPAM</h4>
+      <p>MAAS integrates best-of-breed ISC DHCP and Bind9 DNS, and will operate these in high availability mode whenever redundant MAAS services are provided.</p>
+      <p>Deploy the MAAS API services on two systems and you will have high availability API endpoints; deploy two rack controllers and all rack services are automatically redundant.</p>
+      <p>MAAS uses the Postgres SQL database which supports redundant and highly available scenarios.</p>
+      <h4>Devops on bare metal is easy with dynamic server provisioning</h4>
+      <p>Take devops to bare metal for apps like big data, kubernetes, analytics, machine learning, private cloud, OpenStack, PAAS and HPC. The specialists love MAAS.</p>
+    </div>
+    <div class="col-6">
+      <h4>Open source software-defined data center</h4>
+      <p>MAAS is an open source SDDC solution used by telcos, financial institutions, media companies and supercomputer admins to take care of all the low-level details.</p>
+      <p>PXE, IPMI, ILO and all the custom protocols needed for diverse vendor hardware support come together in one clean REST API with Python bindings for easy integration and automation.</p>
+      <p>It includes a full IPAM capabilities, providing a central database and REST API for all network addressing and naming information.</p>
+      <h4>KVM MicroCloud for Edge Computing</h4>
```

Comment:
We used "KVM Micro-cloud" earlier. Which is it?

Reply:
Opted to go with KVM MicroCloud. @cassiocassio to confirm?